openedx/core/djangoapps/content/block_structure/tests/test_signals.py

```diff
@@ -47,15 +46,6 @@ def test_course_update(self):
 
     @ddt.data(True, False)
     @patch('openedx.core.djangoapps.content.block_structure.manager.BlockStructureManager.clear')
```

Comment:
The above two lines should also be removed, since they are specific decorators for this test.